api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/ClaimServiceTest.java

```diff
@@ -306,9 +300,9 @@ private void shouldSendAdditionalPregnancyPaymentMessage(Claimant existingClaima
 
         //then
         assertThat(result.getClaimUpdated()).isTrue();
-        assertThat(result.getUpdatedFields()).isEqualTo(singletonList(EXPECTED_DELIVERY_DATE.getFieldName()));
-        AdditionalPregnancyPaymentMessagePayload expectedPayload = AdditionalPregnancyPaymentMessagePayload.builder().claimId(existingClaimId).build();
-        verify(messageQueueClient).sendMessage(expectedPayload, ADDITIONAL_PREGNANCY_PAYMENT);
+        List<String> updatedFields = singletonList(EXPECTED_DELIVERY_DATE.getFieldName());
+        assertThat(result.getUpdatedFields()).isEqualTo(updatedFields);
```

Comment:
I'd be tempted here to use `containsExactly` rather than `isEqualTo` as you then don't need to build the list above.